back/config/initializers/1_locales.rb

```diff
@@ -2,17 +2,59 @@
 
 require 'i18n/backend/fallbacks'
 
-CL2_SUPPORTED_LOCALES = %i[en ar-MA ar-SA en-GB en-CA nl-BE nl-NL fr-BE fr-FR de-DE it-IT
-                           da-DK nb-NO es-ES es-CL pl-PL hu-HU kl-GL lb-LU ro-RO pt-BR mi sr-Latn sr-SP hr-HR].freeze
+CL2_SUPPORTED_LOCALES = %i[
+  en
+  ar-MA
+  ar-SA
+  en-GB
+  en-CA
+  nl-BE
+  nl-NL
+  fr-BE
+  fr-FR
+  de-DE
+  it-IT
+  da-DK
+  nb-NO
+  es-ES
+  es-CL
+  pl-PL
+  hu-HU
+  kl-GL
+  lb-LU
+  ro-RO
+  pt-BR
+  mi
+  sr-Latn
+  sr-SP
+  hr-HR
+].freeze
 
-I18n.available_locales = CL2_SUPPORTED_LOCALES + %i[nl fr de it da nb es pl hu kl lb ro pt ar sr hr]
+FALLBACK_LOCALES = %i[
```

Comment:
If `FALLBACK_LOCALES` constant is not used outside of this file, maybe it makes sense to make it a local variable? 
So, if we want to refactor it again, we don't need to check it globally (which can be tricky because of sth like `constantize("#{name}_LOCALES")`.

Reply:
Mh, good point. Maybe I can use `private_constant` as well, to have best of both world? Not sure how that behaves, because it's not scoped to a module or class, I'll give a try!